### PR TITLE
feat(ui): init & app list page

### PR DESCRIPTION
**Pull request by shhdgit**
- init ui workflow
- init brand new app list page
image

### CODE REVIEW

File: ui/src/modules/app_list/index.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  )
+}
+
+const AppCard: React.FC<{ app: ApplicationInfo }> = ({ app }) => {
+  const createdAt = useMemo(() => dayjs.unix(app.created_at).format('MMM D, YYYY'), [app])
+  const timeFromNow = useMemo(() => dayjs.unix(app.created_at).fromNow(), [app])
+
+  const openDeleteModal = () =>
+    modals.openConfirmModal({
+      title: <Title order={5}>Delete App Name</Title>,
```
Comment (freedeaths) [inline ui/src/modules/app_list/index.tsx:101]: Delete Application or Delete {app name}?

File: ui/src/modules/app_list/index.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        <Text size="sm">
+          Deleting the application may cause online malfunctions. Confirm to delete the application?
+        </Text>
+      ),
+      centered: true,
+      size: 'lg'
+    })
+
+  return (
+    <Card onClick={() => console.log(123)}>
```
Comment (freedeaths) [inline ui/src/modules/app_list/index.tsx:116]: I think we should delete it.

File: ui/dev.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+COPY package.json .
+COPY yarn.lock .
+RUN yarn
+VOLUME [ "/app/node_modules" ]
+
+RUN echo "#!/bin/sh" >> /entry_point.sh
+RUN echo "yarn dev --host" >> /entry_point.sh
+RUN chmod +x /entry_point.sh
+
+CMD ["/entry_point.sh"]
```
Comment (lucklove) [inline ui/dev.Dockerfile:13]: The entry_point.sh only contains one line, what about just `CMD [ "yarn", "dev", "--host"]`?

Comment (shhdgit) [reply ui/dev.Dockerfile:13]: make sense!

**Review by lucklove — APPROVED** [review]
LGTM

**Pull request merged**